components/match2/wikis/rocketleague/match_group_input_custom.lua

```diff
@@ -203,10 +203,17 @@ end
 function matchFunctions.getLinks(match)
 	match.links = {}
 
-	-- Octane
-	match.octane1 = match.octane1 or match.octane
-	for key, octane in Table.iter.pairsByPrefix(match, 'octane') do
-		match.links[key] = 'https://octane.gg/matches/' .. octane
+	-- Shift (formerly Octane)
```

Comment:
i personally would prefer a bot job instead

Reply:
Shouldn't be necessary in my opinion